src/config/sidebar.ts

```diff
@@ -216,15 +216,45 @@ export const SIDEBAR: Partial<Record<Sections, SectionEntry[]>> = {
           url: "data-streams/getting-started-hardhat",
         },
         {
-          title: "Stream IDs",
+          title: "Data Streams Feed IDs",
           url: "data-streams/stream-ids",
         },
+        {
+          title: "Billing",
+          url: "data-streams/billing",
+        },
         {
           title: "Release Notes",
           url: "data-streams/release-notes",
         },
       ],
     },
+    {
+      section: "Guides",
+      contents: [
+        {
+          title: "Fetch and decode reports via a REST API",
+          url: "data-streams/tutorials/streams-direct-api",
+        },
+        {
+          title: "Stream and decode reports via WebSocket",
+          url: "data-streams/tutorials/streams-direct-ws",
```

Comment:
```suggestion
          title: "Fetch and decode reports (REST API)",
          url: "data-streams/tutorials/streams-direct-api",
        },
        {
          title: "Stream and decode reports (WebSocket)",
          url: "data-streams/tutorials/streams-direct-ws",
```

Optional change: Just saving a few characters in the navigation bar. The page title doesn't need to match exactly.